integrations/drift/drift.go

```diff
@@ -90,7 +90,6 @@ func (api driftAPIProduction) request(method string, endpoint string, payload []
 	req.Header.Set("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	defer resp.Body.Close()
```

Comment:
That's a possibility. I'll look into it and see if this is possible. If that is the case it would probably be a better solution than the current one.